src/main/java/org/takes/facets/auth/codecs/CcAES.java

```diff
@@ -144,6 +116,27 @@ public Identity decode(final byte[] bytes) throws IOException {
         }
     }
 
+    private static AlgorithmParameterSpec createAlgorithmParameterSpec() {
+        final SecureRandom random = new SecureRandom();
+        final byte[] bytes = new byte[CcAES.BLOCK];
+        random.nextBytes(bytes);
+        return new IvParameterSpec(bytes);
+    }
+
+    /**
+     * A private method to check the block size of the key.
```

Comment:
@mkordas I use the method twice. What are the other options?
The previous implementation, store this value in the field in the constructor creation.